test/rpc_test.py

```diff
@@ -21,7 +21,6 @@
 
 
 BACKEND = getenv("RPC_BACKEND", RpcBackend.PROCESS_GROUP)
```

Comment:
nit, change this name to distinguish between RpcBackend and communication backends (gloo, nccl, mpi)?

Reply:
@rohan-varma 

I would love to, but I will not do it in this PR.
For reducing changes and chance to break in a single PR.